ui-components/src/locales/general.json

```diff
@@ -378,6 +378,49 @@
         "legend": "Housing vouchers, nontaxable income or rental subsidies"
       }
     },
+    "programs": {
+      "servedInMilitary": {
+        "summary": "Veteran of the US Military",
+        "servedInMilitary": {
+          "label": "Yes"
+        },
+        "doNotConsider": {
+          "label": "No"
+        }
+      },
+      "tay": {
+        "summary": "Transition Age Youth (TAY)",
+        "tay": {
+          "label": "Yes"
```

Comment:
Ideally we don't have duplicate translation strings, is there a way to prevent that here?

Reply:
Probably not for the "Yes" options since they could be keyed anything, but the No options are always keyed "doNotConsider" so I could DRY that up.

Reply:
If for programs there are only ever two answers, couldn't on the frontend we always use the "yes" label for the first key?

Reply:
I was hoping that would be the case, but the homeless one is an oddball, it's not Yes/No answers. 🤷‍♂️

Reply:
Ahhh okay no worries